docs/coolit-guide.md

```diff
@@ -0,0 +1,51 @@
+# Corsair Hydro H110i GT AIO liquid cooler
+_Driver API and source code available in [`liquidctl.driver.coolit`](../liquidctl/driver/coolit.py)._
+
+## Initialization
+[Initialization]: #initialization
+
+The AIO does not need to be initialized prior to use, but it will set the pump
+mode to quiet.
```

Comment:
If _quiet_ is in backticks later on, maybe surround it here as well.

Reply:
Corrected